packages/webpack/src/utils/style-loader.js

```diff
@@ -25,6 +26,16 @@ export default class StyleLoader {
     return Boolean(this.isServer && this.extractCSS)
   }
 
+  isUrlResolvingEnabled (url, resourcePath) {
+    // Ignore absolute URLs, it will be handled by serve-static.
+    if (url.startsWith('/')) {
+      consola.warn(`Please use relative path or alias path instead of absolute path ${url}`)
```

Comment:
Absolute url works when we ignore it from css-loader, but I think we'd better put the warning message here for letting user use relative or alias path